services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -27,7 +27,7 @@ def _docker_compose_options_from_settings(settings: ApplicationSettings) -> str:
 
 
 def _pad_docker_command_timeout(
-    docker_command_timeout: Optional[int], *, padding: int = 2
+    docker_command_timeout: Optional[int], *, padding: int = 10
```

Comment:
suggestion: padding, do you mean delay?

Reply:
and why is it changed to 10? this looks suspicious.

Reply:
So some commands like the docker-compose down (have a default timeout of 10 second). I tried adding a "padding" some more delay to allow the command to finish. But I put in a very random number 2 seconds. This would result in a timeout. So I raised this number. I am open to suggestion for a better name.
Maybe I call the function `_get_increaased_docker_command_timeout` and the argument would be `timeout_increase`?

Reply:
refactored this and renamed function